### DISCUSSION TITLE
I have installed few softwares like along with main installer package(though main installer package to be more precise) through bundle.wxs in wix bootstrapper application as mentioned in description.Please provide /suggest way to uninstall it just using the way they are installed

### DISCUSSION THREAD

**Discussion by vaa2000** [wixtoolset/issues#8622] [Questions]
I have installed few softwares like along with main installer package(though main installer package to be more precise) through bundle.wxs in wix bootstrapper application as mentioned in description.Please provide /suggest way to uninstall it just using the way they are installed .Python Launcher, 
Codemeter Kit v7.40b,
Microsoft Visual C++ 2015-2022 Redistributable (x86) - 14.32.31332,
Microsoft Visual C++ 2015-2022 Redistributable (x64) - 14.32.31332,
Python 3.11.1 (64-bit) As a workaround I have written powershell script to uninstall them by calling registry entries for each software, however after calling this powershell script through customactions still these 5 softwares are not getting uninstalled

**Comment by bevanweiss** [comment]
This really seems like a question you should be asking the authors of those 3rd party programs.
"How do I uninstall your software in an unattended manner?"

Nothing about your request sounds like a WiX toolset specific problem.
WiX Toolset is just a wrapper around Windows Installer functionality (and some additional 'bonus' functionality for situations where Windows Installer doesn't quite stretch).

For your Microsoft VC Runtime items, Microsoft provide Merge Modules (msm) files for these, depending on versions it may or may not be the Microsoft recommended method of distributing.
https://learn.microsoft.com/en-us/cpp/windows/redistributing-components-by-using-merge-modules
